spec/smarty_streets_spec.rb

```diff
@@ -75,21 +96,62 @@
           location.zipcode = '95014'
         end
 
-        expect(locations.first.street).to  eq '1 Infinite Loop'
-        expect(locations.first.city).to    eq 'Cupertino'
-        expect(locations.first.state).to   eq 'CA'
+        expect(locations.first.street).to eq '1 Infinite Loop'
+        expect(locations.first.city).to eq 'Cupertino'
+        expect(locations.first.state).to eq 'CA'
         expect(locations.first.zipcode).to eq '95014-2083'
       end
     end
 
     context 'address with no candidates' do
-      let(:response_code) { 200 }
-      let(:candidates) { 5 }
+      before do
+        allow(Net::HTTPResponse).to receive(:code)
+          .and_return(200)
+
+        allow(Net::HTTPResponse).to receive(:body)
+          .and_return(nil)
+      end
 
       specify 'makes a request for a standardized address' do
-        expect {
-          locations = SmartyStreets.standardize {}
-        }.to raise_error SmartyStreets::Request::NoValidCandidates
+        expect do
+          SmartyStreets.standardize {}
+        end.to raise_error SmartyStreets::Request::NoValidCandidates
+      end
+    end
+
+    context 'Connection Timeout' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.